### ISSUE TITLE
Remove unnecessary spaces between numbers, format the list properly on the "How to get free language assistance from VA" page

### ISSUE THREAD

**Issue by artsymartha68** [department-of-veterans-affairs/va.gov-team#139261]
These two accessibility bugs were reported by the VA 508 Office.

Bugs are found in the content on this page: https://www.va.gov/resources/how-to-get-free-language-assistance-from-va/

**Bug 1: There should be no spaces between the phone numbers in the two aria labels. The spaces are read aloud by JAWS.**

`<p>Call our MyVA411 main information line at <a aria-label="8 0 0. 6 9 8. 2 4 1 1." href="tel:+18006982411">800-698-2411</a> (TTY: <a aria-label="7 1 1." href="tel:711">711</a>). Select 0 to connect with a VA call center agent. Tell the agent you’d like an interpreter to join the call. We’re here 24 hours a day, 7 days a week.</p>`

**Bug 2: The bulleted items are not properly formatted as a list using `<ul>` and `<li>` tags. This is incorrect code and as such doesn't read aloud as a list in JAWS.**

```
<p>Your patient advocate can help you get services like these:</p>
 
<p>&#x2022;&#xA0;&#xA0; &#xA0;An interpreter<br>
&#x2022;&#xA0;&#xA0; &#xA0;Translated documents to help you understand some VA health benefits and services<br>
&#x2022;&#xA0;&#xA0; &#xA0;American Sign Language services</p>
```

**Comment by RLHecht** [comment]
@laurenvocke-va Could you review my changes on this page and publish? I removed the links from the phone number and TTY and fixed the way we had the bullets set up. I think that should address these 2 issues. Thanks!

**Comment by laurenvocke-va** [comment]
Reviewed and published. I'll confirm when it's live

**Comment by laurenvocke-va** [comment]
Changes are live! Closing this ticket

**Comment by artsymartha68** [comment]
Hot damn, y'all are awesome. Thanks!

**Comment by RLHecht** [comment]
@artsymartha68 would you be able to check that the fixes we made solve the issue? I think they should since this page was first created so long ago and set up a little differently than how we do it now but i just want to make sure it actually fixed the bug!

**Comment by artsymartha68** [comment]
I'll ask our VA 508 office colleague to verify.

**Comment by artsymartha68** [comment]
Verified: fixed. Thanks again!
